ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_manipulation.py

```diff
@@ -1015,3 +1015,47 @@ def test_expand(
         x=x[0],
         shape=shape,
     )
+
+
+@st.composite
+def _concat_from_sequence_helper(draw):
+    dtypes, arrays, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("valid"),
+            num_arrays=helpers.ints(min_value=1, max_value=6),
+            ret_shape=True,
```

Comment:
I think setting `shared_dtype=True` will fix the tensorflow error I see in the CI test. It seems tensorflow can only concat arrays of a common dtype.

Reply:
okay, I will try doing that as well.